src/components/icons/_icons.scss

```diff
@@ -1,6 +1,61 @@
 $includeHtml: false !default;
 
 @if ($includeHtml) {
+  .mint-icon {
+    fill: $white;
+    width: 26px;
```

Comment:
@kdzwinel found out this is one of the most commons sizes used currently on web page :) 
We will refactor it to variables.